src/c_kzg_4844.c

```diff
@@ -1041,22 +1042,11 @@ static C_KZG_RET compute_kzg_proof(KZGProof *out, const Polynomial p, const BLSF
   return C_KZG_OK;
 }
 
-typedef struct {
-    unsigned int h[8];
-    unsigned long long N;
-    unsigned char buf[64];
-    size_t off;
-} SHA256_CTX;
-
-void sha256_init(SHA256_CTX *ctx);
-void sha256_update(SHA256_CTX *ctx, const void *_inp, size_t len);
-void sha256_final(unsigned char md[32], SHA256_CTX *ctx);
```

Comment:
On OSX, we have `llvm-objcopy` -- but it doesn't support `--globalize-symbol` for Mach0 files.

Attempting to use `llvm-objcopy --globalize-symbol` on a Mach0 errors:
 /usr/local/opt/llvm/bin/llvm-objcopy: error: option is not supported for MachO

I copy-pasted a [different SHA256 implementation](https://github.com/B-Con/crypto-algorithms/blob/master/sha256.c). We should probably do something other than edit the library to expose BLST's internal SHA functions.

Maybe we could use OpenSSL's? https://github.com/openssl/openssl/blob/master/crypto/sha/sha256.c

Reply:
@xrchz let me know what you prefer here.

Reply:
Can we patch the blst submodule to expose the functions? I think you just need to remove the `static` declaration from those functions in `sha256.h`.

Reply:
Yes that works